beacon-chain/sync/rpc_metadata.go

```diff
@@ -70,7 +70,9 @@ func (s *Service) metaDataHandler(_ context.Context, _ interface{}, stream libp2
 	switch streamVersion {
 	case p2p.SchemaVersionV1:
 		switch metadataVersion {
-		case version.Altair, version.Deneb:
+		case version.Altair, version.Fulu:
```

Comment:
why was deneb removed 🤔

Reply:
Ohhhh maybe yes.
At the very beginning, peerDAS had to be IN deneb, right. (Like activated at a given epoch, but without a hark fork.)
So, yes, that's probably why I wrote that a long time ago.